pages/calendar/index.tsx

```
@@ -589,186 +585,162 @@ const CalendarView: FC<ICalendarViewProps> = ({ className, small = false, emitCa
 	];
 
 	return (
-		<div className='w-full overflow-auto'>
-			<div
-				className={`${className} ${categoriesLoading ? 'w-auto' : 'w-max'} rounded-xl bg-white p-3 drop-shadow-md dark:border-separatorDark dark:bg-section-dark-overlay md:w-auto`}
-			>
+		<>
+			<div className={`${className} mb-5 rounded-xl bg-white p-3 drop-shadow-md dark:border-separatorDark dark:bg-section-dark-overlay`}>
+				<Spin
+					spinning={categoriesLoading}
+					indicator={<></>}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)